code/modules/paperwork/paper.dm

```diff
@@ -108,220 +147,87 @@
 	user.visible_message("<span class='suicide'>[user] scratches a grid on [user.p_their()] wrist with the paper! It looks like [user.p_theyre()] trying to commit sudoku...</span>")
 	return (BRUTELOSS)
 
+/// ONLY USED FOR APRIL FOOLS
+
 /obj/item/paper/proc/reset_spamflag()
 	spam_flag = FALSE
 
 /obj/item/paper/attack_self(mob/user)
-	user.examinate(src)
+	if(edit_mode != MODE_READING)
+		log_paper("<span class='warning'>Not sure why, but there is an open ui and its in his hand?</span>")
```

Comment:
My brain is too small to understand this message

Reply:
I misinterpreted what attack self does hah